src/libraries/System.Net.Quic/src/System.Net.Quic.csproj

```diff
@@ -83,6 +121,14 @@
     <Reference Include="System.Threading.Channels" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Security.Cryptography.OpenSsl\src\System.Security.Cryptography.OpenSsl.csproj" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetsUnix)' == 'true'">
+    <Reference Include="System.Diagnostics.StackTrace" />
```

Comment:
Extra? If anything I would add it to the Debug only addition for Console. perhaps as separate PR?

Reply:
Not extra, used by one the added interop sources:
https://github.com/dotnet/runtime/blob/b937677e8f8601848d29bc072a93cc0c6e21576d/src/libraries/Common/src/Microsoft/Win32/SafeHandles/SafeX509Handles.Unix.cs#L16-L16

But you're right it's only for Debug, I'll fix that.

Reply:
Fixed.